source/documentation/make-updates-during-tax-year.html.md

```diff
@@ -153,11 +153,19 @@ If a customer wants to make a change to a previously submitted periodic update,
 3. Software prepares the summary information and displays it to the customer.
 4. The customer checks the information and submits it.
 5. The software initiates the data submission process by calling one of the following endpoints, depending on the income source type being submitted:
+<<<<<<< HEAD
```

Comment:
Please remove this, thanks. Seems to be an issue when conflicts were resolved. See screenshot below

<img width="1160" alt="Screenshot" src="https://github.com/user-attachments/assets/798df165-7514-4a3a-a0d2-8cfc3966b2f8" />